pkg/repository/v1/scheduler_rate_limit.go

```diff
@@ -17,76 +19,65 @@ func newRateLimitRepository(shared *sharedRepository) *rateLimitRepository {
 }
 
 func (d *rateLimitRepository) ListCandidateRateLimits(ctx context.Context, tenantId pgtype.UUID) ([]string, error) {
-	// TODO: FIX ME
-	return []string{}, nil
-}
-
-func (d *rateLimitRepository) UpdateRateLimits(ctx context.Context, tenantId pgtype.UUID, updates map[string]int) (map[string]int, error) {
-	// TODO: FIX ME
-	return map[string]int{}, nil
-}
-
-// TODO: ADD RATE LIMITS
-// func (d *rateLimitRepository) ListCandidateRateLimits(ctx context.Context, tenantId pgtype.UUID) ([]string, error) {
-// 	rls, err := d.queries.ListRateLimitsForTenantNoMutate(ctx, d.pool, dbsqlc.ListRateLimitsForTenantNoMutateParams{
-// 		Tenantid: tenantId,
-// 		Limit:    10000,
-// 	})
+	rls, err := d.queries.ListRateLimitsForTenantNoMutate(ctx, d.pool, sqlcv1.ListRateLimitsForTenantNoMutateParams{
+		Tenantid: tenantId,
+		Limit:    10000,
```

Comment:
nit: magic number? is there any real risk or someone exceeding this?

Reply:
Declared as a constant at the top of the file -- this is the max number of rate limits we can support per tenant, though it's not documented and we don't enforce it everywhere.

Reply:
great, thanks!